database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -210,20 +232,29 @@ private String getJDBCPluginId() {
 
     @Description("The SELECT query to use to import data from the specified table. " +
       "You can specify an arbitrary number of columns to import, or import all columns using *. " +
-      "The Query should contain the '$CONDITIONS' string. " +
+      "The Query should contain the '$CONDITIONS' string unless numSplits is set to one. " +
       "For example, 'SELECT * FROM table WHERE $CONDITIONS'. The '$CONDITIONS' string" +
       "will be replaced by 'splitBy' field limits specified by the bounding query.")
     String importQuery;
 
+    @Nullable
     @Name(BOUNDING_QUERY)
     @Description("Bounding Query should return the min and max of the " +
-      "values of the 'splitBy' field. For example, 'SELECT MIN(id),MAX(id) FROM table'")
+      "values of the 'splitBy' field. For example, 'SELECT MIN(id),MAX(id) FROM table'. " +
+      "This is required unless numSplits is set to one.")
     String boundingQuery;
 
+    @Nullable
     @Name(SPLIT_BY)
-    @Description("Field Name which will be used to generate splits.")
+    @Description("Field Name which will be used to generate splits. This is not required if numSplits is set to one.")
```

Comment:
nit: May be we should use the same language as in other description. 'This is required unless numSplits is set to one'.